css-overflow-5/Overview.bs

```diff
@@ -288,33 +288,51 @@ The 'scroll-marker-group' property</h4>
 	This 'eventual scroll position' is used to determine the active marker within each [=scroll marker group=].
 	Since markers themselves may represent just the start of the content (e.g. headers), we consider the active marker to be the first one which we are at or beyond the scroll position of.
 
-	<div algorithm="scroll tracking">
-		Whenever a [=scroll container=] is scrolled, or layout changes the scroll position, the user agent must run these steps to determine the active marker:
+	Whenever a [=scroll container=] participating in one or more [=scroll marker groups=] is scrolled, or layout changes the eventual scroll position,
+	the user agent should determine and update the active marker for each [=scroll marker group=]
+	based on the section of content most likely to be of interest given the target scroll position.
+
+	<div algorithm="scroll tracking" class=example>
+		Example algorithm to determine the active marker for a given scroll marker <var>group</var>.
+
+		1. Let |scroller| be the nearest common ancestor [=scroll container=] of all of the [=scroll marker=] elements in |group|.
+		1. Let |active| be scroller.
+		1. While |active| is a [=scroll container=] containing [=scroll target=] elements targeted by |group|:
+			1. Let <var>scroller</var> be |active|.
+			1. Let <var>targets</var> be the set of the [=scroll target=] elements whose nearest ancestor [=scroll container=] is |scroller|
+			   and the [=scroll container=] elements which contain [=scroll target=] elements targeted by the [=scroll marker group=] whose nearest ancestor [=scroll container=] is |scroller|.
+			1. Let <var>primary</var> be the primary scrolling axis, assumed to be the block direction of the container's writing-mode.
+			1. Let <var>secondary</var> be the scrolling axis perpendicular to primary.
+			1. Let <var>position</var> be the 'eventual scroll position' considering ongoing scrolling operations.
+			1. For each <var>axis</var> of |primary|, followed by |secondary|:
+				1. 	Let <var>scrollport size</var> be the client size of |scroller| in the dimension |axis|.
```

Comment:
We probably want the size here and the positions used to account for the [optimal viewing region](https://drafts.csswg.org/css-scroll-snap/#optimal-viewing-region), though perhaps this is a followup?